### ISSUE TITLE
Revise fluid scale because vw and vh cause text to fail WCAG zoom requirements

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/c3-eleventy#23]
* https://adrianroselli.com/2019/12/responsive-type-and-zoom.html
* https://www.w3.org/WAI/WCAG21/Techniques/failures/F94.html

The text doesn't scale linearly because of the `--fluid-screen: 100vw` declaration in **_variables.css**. Will need to find another means to scale text. Screenshots below show text at 100% and 400%, 1280px width in Chrome Big Sur.

A better way might be Andy Bell's Fluid Typography with CSS Clamp ( https://piccalil.li/tutorial/fluid-typography-with-css-clamp/ ). Will investigate in a CodePen and circle back.

---
**100% (default)**
<img width="1158" alt="Screen Shot 2021-06-22 at 7 21 10 AM" src="https://user-images.githubusercontent.com/934879/122924481-6a92eb80-d32b-11eb-9460-a23396fa06da.png">

---
**400% zoom**
<img width="1088" alt="Screen Shot 2021-06-22 at 7 23 21 AM" src="https://user-images.githubusercontent.com/934879/122924527-78e10780-d32b-11eb-8773-88720fc6c99b.png">

**Comment by 1Copenut** [comment]
Finished this (for now) with commit https://github.com/1Copenut/c3-eleventy/commit/53ffa43fbd43745ba3e0e16188e6125e0dbaff52
